services/drupal/web/themes/epa_theme/source/_patterns/03-uswds/site-alert/_site-alert.scss

```diff
@@ -0,0 +1,25 @@
+@import 'components/site-alert';
+
+.usa-site-alert {
+  .usa-alert {
+    margin-bottom: 0;
+    overflow: visible;
+  }
+
+  .usa-alert__text {
+    > :last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  .usa-alert__body {
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
@kmonahan What is this used for? It currently puts the title and content on the same line.

![uswds-site-alert](https://user-images.githubusercontent.com/135259/121262343-f44dae00-c881-11eb-96b5-8451ed270600.jpg)

Reply:
The Close button

Reply:
It's really just changing the class selector on https://github.com/USEPA/webcms/blob/main/services/drupal/web/themes/epa_theme/source/_patterns/05-components/sitewide-alert/_sitewide-alert.scss#L16-L19

Reply:
Ah, is this for the close button on site alerts that can be closed?

Reply:
Yep, exactly.